packages/@glimmer/validator/index.ts

```diff
@@ -1,3 +1,17 @@
+import { symbolFor, getGlobal } from './lib/utils';
+
+const globalObj = getGlobal();
+
+const GLIMMER_VALIDATOR_REGISTRATION = symbolFor('GLIMMER_VALIDATOR_REGISTRATION');
+
+if ((globalObj as any)[GLIMMER_VALIDATOR_REGISTRATION] === true) {
```

Comment:
This seems to be the best way to do this, we can't actually add it as an interface because `Symbol.for()` does not produce a `unique symbol`, so it can't be used as a key in an interface.